tjreports/site/models/reports.php

```diff
@@ -1122,12 +1166,13 @@ private function filterReportColumns($queryId, &$selColToshow)
 	 * @param   ARRAY  &$showhideCols  Show Hide columns
 	 * @param   ARRAY  &$colToshow     Columns to show
 	 * @param   ARRAY  &$selColToshow  Selected Cols
+	 * @param   ARRAY  &$piiColumns    Personally Identifiable Information column
 	 *
 	 * @return  Void
 	 *
 	 * @since   3.0
 	 */
-	private function processSavedReportColumns($queryId, &$showhideCols, &$colToshow, &$selColToshow)
+	private function processSavedReportColumns($queryId, &$showhideCols, &$colToshow, &$selColToshow, &$piiColumns)
```

Comment:
can we not use this as `$this->piiColumns` instead of passing as param?

Reply:
Komal is working on this. only queryId will be passed to fuction other variables will be class variables